Jenkinsfile

```
@@ -1,8 +1,9 @@
 node {
     checkout scm
     stage("Test") {
-        docker.image('postgres:11').withRun('-e POSTGRES_PASSWORD=mysecretpassword') { c ->
-            docker.image('postgres:11').inside("-e PGPASSWORD=mysecretpassword -e PGHOST=postgres -e PGUSER=postgres --link ${c.id}:postgres") {
+    	def pwd = "secret"
+        docker.image('postgres:11').withRun("-e POSTGRES_PASSWORD=${pwd}") { c ->
```

Возможно, что все переменные среды нужно выставлять в первом контейнере, которые будут наследоваться отлинкованным контейнером?

```suggestion
        docker.image('postgres:11').withRun("-e PGPASSWORD=${pwd} -e POSTGRES_PASSWORD=${pwd}") { c ->
```